### ISSUE TITLE
Please restore prior Run Mode description in Readme.txt

### ISSUE THREAD

**Issue by w7sst** [w7sst/MorseRunner#284]
### Description
Hi Scott,
I think we should restore the original descriptions of the Run Modes into the `Readme.txt` file. As I was setting up the default Run Modes with Start/Stop button changes, I noticed that these descriptions had been removed. I think they are important to keep around since they describe the `Competition` and `HST Competition` modes and the conditions provided by each. 

The 1.83 `Readme.txt` file contained:
```
  The selected contest can be started in one of four modes.

    Pile-Up mode: a random number of stations calls you after you send a CQ. Good
    for improving copying skills.

    Single Calls mode: a single station calls you as soon as you finish the
    previous QSO. Good for improving typing skills.

    WPX Competition mode: similar to the Pile-Up mode, but band conditions and contest
    duration are fixed and cannot be changed. The keying speed and band activity
    are still under your control.

    HST Competition mode: all settings conform to the IARU High Speed Telegraphy
    competition rules.
```
I think we should somehow keep the description of these four modes in our release notes.

Bob has been asking us to restore the original 1.68 behaviors around HST, so we will probably want to keep this in there. I am leaning towards either
1. letting the `HST Competition` Run Mode force selection of `HST (High Speed Test)` Contest and forcing all other settings (similar to how 1.68 forces several settings.
2. Requiring the user to first select `HST (High Speed Test)` Contest before the `HST Competition` Run Mode is available.

I'm leaning towards the first choice above, but haven't discussed this choice with anyone.

Thoughts?

**Comment by w7sst** [comment]
Hi @scotthibbs ,
We also need to mention the changes to the Start/Stop button as covered in #282

**Comment by scotthibbs** [comment]
Obviously, I liked the rewording of the run button because I changed it. :) hihi 
I changed it to this: 
  The Run Button
    This starts and stops the competition or contest.
    Starting a Contest -
      To start the contest that is selected in the contest drop down,
      set the duration in the minutes box and use the run drop down
      to select the desired mode for the contest (single call or pile up).
      You may have to hit F1 (or Enter) to start sending CQ.

    Starting a Competition -
      To start a Competition, select either the WPX and HST competitions found
      in the run drop down. Many settings are locked for the competitions but
      The keying speed and activity are still under your control.
      The default duration of a competition session is 60 minutes. You can
      change this setting to any time desired. To set a new default
      use the CompetitionDuration entry in the MorseRunner.ini file, e.g.:
        [Contest]
        CompetitionDuration=15
        
I kept most of the descriptions/wording for these modes and competitions. There were several reasons for the change. I felt the previous list wasn't user friendly, just descriptions. This previous list is from 1.68 which did not have a contest dropdown. I felt we needed to point out how run button menu is a function (single call or pile up) of the contest drop down menu. So we state that the user can run the contest previously selected or a competition. The document now flows from the users standpoint working down from the right of the gui to the run button (an extended how to get started). I also tried not reintroducing topics for better readability. Thus I pointed out the CompetitionDuration setting without having to reintroduce the topic in the "Additional Settings".    

Restoring the 1.68 HST competition. I am a big fan of this. 

It's important to note that we created some confusion (even amongst ourselves) about the difference between a contest and competition, since we have added the "contests" dropdown to this program. Previously the only "contests" we had from 1.68 were competitions found in the run buttion where settings were locked. My suggestion to solve this confusion is simple: Move the competitions (where settings are force selected) to the contest drop down. "Original HST Competition" can be one of these which was an hour, and we can add "IARU HST Competition" which I think is ten minutes using 1.67. (Alternatively we can select a choice between competition and contest for the dropdown selections. Thus each "contest" can have a contest with locked settings for 15 minutes. This helps later with reporting results on a webpage) In the same turn, I would like to see a choice under the contest dropdown for the mode (pile up or single call). To sum up, my suggestion is removing the run button menu. 

Since this is a created issue, I'm just going to make this comment and leave it be for you. I like my arrangement as stated above but if you would like to tweak it please respond with your changes. I write the readme and then it gets edited by the community which is a good thing. 
Scott

**Comment by w7sst** [comment]
Hi Scott,
You raise a lot of good points here. Yes, this is an area where there is confusion. I was certainly confused at first when I started adding contests and I admit I was ignoring the Run button and its underlying run modes.

Here is my summary of the differences between Contests and Run Mode.

__Contests__ tend to have
- a set of defined rules
- a specific exchange
- an associated call history file

__Run Mode__ tends to
- define the conditions under which a Contest us executed. Conditions could include band conditions (the various checkboxes in the Band Conditions group on existing UI), duration, restrictions on Serial NR, CW Rx Speed adjustments etc. An example being HST Competition locks everything down to match the IARU High Speed Telephony contest rules.

__Competitions__ ...
- A good way to compare scores against other users.
- Applies a common set of rules/conditions so the resulting score can be compared.
- Any contest could be executed under a given Competition umbrella.

In MRCE, we are adding many contests. As you have mentioned, any contest can be executed under various runtime conditions (Single Call, Pile Up, and various forms of Competition). We are looking for a way to clearly separate the two concepts.

The HST Competition is actually using the CQ WPX Contest under the hood, with some restrictions applied.

For me, I've realized that MRCE will have multiple Contests and it will have multiple Run Modes/Competitions available. We will continue this conversation as we sort through this.

**Comment by w7sst** [comment]
Scott,
I'm sorry and want to apologize. When I submitted this Issue, I was in a hurry and didn't explain myself very well. Let me try again...

I guess the part that I was hoping we could restore was the description of the actual conditions as described in `WPX Competition` and `HST Competition`. Which is...
> WPX Competition mode: __similar to the Pile-Up mode, but band conditions and contest
> duration are fixed and cannot be changed. The keying speed and band activity
> are still under your control.__
>
> HST Competition mode: __all settings conform to the IARU High Speed Telegraphy
> competition rules.__

I guess all I was asking for was to restore the descriptions of the conditions simulated/imposed by the two competitions. After all, this is what makes them different from each other. For example, in 1.68:
- WPX Competition - forces __on__ all Band Conditions (QRM, QRN, QRB, Flutter, LIDs).
- HST Competition - force __off__ all Band Conditions (QRM, QRN, QRB, Flutter, LIDs).

By dropping the descriptions, we lost Alex's original intent of these two modes.

As we think through the notions of a Competition, we will be defining what makes them different and then allow any Contest to be executed within a Competition. We'll get it figured out.

**Comment by scotthibbs** [comment]
ahhh.. gotcha.. Ok. tomorrow night I'll jump back into the readme and update the differences in the competitions. They do need to be explained better. 

Agree with your definitions of the competitions (forced settings) and want to add a set time limit and would suggest 15 minutes. (Seems like committing to half an hour or more of a competition is pushing it but 15 minutes is "do-able") By setting the time limit we can compare scores. Agree we may need the gui to have a selection for contest or competition to separate them.

**Comment by scotthibbs** [comment]
I forgot to close this. I previously added the discriptions so that the run button section looks like this:

**Comment by w7sst** [comment]
Awesome! Thank you!
